contracts/libraries/LibUtils.sol

```diff
@@ -59,6 +59,33 @@ library LibUtils {
         }
     }
 
+    /// @notice Verifies that a signature belongs to a given public key
+    /// @param message the message that was signed
+    /// @param publicKey the public key that was used to sign the message
+    /// @param v signature element
+    /// @param r signature element
+    /// @param s signature element
+    /// @return true if the signature is valid, false otherwise
+    function verifySignature(
+        bytes calldata message,
+        bytes calldata publicKey,
```

Comment:
Let's change publicKey to `storage` because we'll always be reading the value from storage on the cursed archaeologist